src/persistence/Elsa.Persistence.EntityFrameworkCore/DbContexts/CustomSchemaModelCacheKeyFactory.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Elsa.Persistence.EntityFrameworkCore.DbContexts
+{
+    public class CustomSchemaModelCacheKeyFactory : IModelCacheKeyFactory
+    {
+        public CustomSchemaModelCacheKeyFactory()
+        {
+        }
+        public object Create(DbContext context)
```

Comment:
I'm noticing some formatting issues here and there. Not a big deal, but perhaps consider using an auto-formatter?